apps/src/maze/maze.js

```diff
@@ -124,10 +153,10 @@ module.exports = class Maze {
     this.PATH_WIDTH = this.SQUARE_SIZE / 3;
   }
 
-  createAnimations(svg) {
+  _createAnimations(svg) {
```

Comment:
I've seen a lot more "private" methods in our codebase with the underscore as a suffix rather than a prefix. Any particular reason to go with the prefix here?

Reply:
Oh, right you are; I'll update them for consistency